nltk/util.py

```diff
@@ -503,35 +496,25 @@ def ngrams(
     :param sequence: the source data to be converted into ngrams
     :type sequence: sequence or iter
     :param n: the degree of the ngrams
-    :type n: int
-    :param pad_left: whether the ngrams should be left-padded
-    :type pad_left: bool
-    :param pad_right: whether the ngrams should be right-padded
-    :type pad_right: bool
-    :param left_pad_symbol: the symbol to use for left padding (default is None)
-    :type left_pad_symbol: any
-    :param right_pad_symbol: the symbol to use for right padding (default is None)
+    :type n: int    
```

Comment:
What editor do you use? It seems to be adding whitespace at the end of lines or at least not removing it. Let me know if you're having trouble getting it to remove such whitespace, I can quickly do it for you.

Reply:
I have been using Atom. I will try to keep a closer eye out for this.

Reply:
This may make your life easier:
https://stackoverflow.com/a/30403353/4501212